https://issues.redhat.com/browse/ACM-17221 Removing KI and adding new… (#7489)
open-cluster-management/rhacm-docs: clusters/release_notes/mce_known_issues.adoc

Learn about known issues and limitations during {mce-short} installation.

[#installnamespace-field-limit]

> === _installNamespace_ field can only have one value

Reviewer: ```suggestion
=== The _installNamespace_ field can only have one value
```

Just to avoid starting with italics

Author: Hi @oafischer . Do you want me to add this as a part of this PR?

Reviewer: I'll leave it up to you, I usually try to start sentences with capitalized words and without formatting

//2.9:ACM-7523

> When enabling the `managed-serviceaccount` add-on, the `installNamespace` field in the `ManagedClusterAddOn` resource must have `open-cluster-management-agent-addon` as the value. Other values are ignored. The `managed-serviceaccount` add-on agent is always deployed in the `open-cluster-management-agent-addon` namespace on the managed cluster.

Reviewer: This needs to go into the cluster section below, not the intro

Author: This is not a part of the [ACM-17221](https://issues.redhat.com/browse/ACM-17221).

Reviewer: OK you can leave it for now then

[#cluster-issues-mce]
== Cluster

Learn about known issues and limitations for Cluster lifecycle with {mce-short}, such as issues with creating, discovering, importing, and removing clusters, and more cluster management issues for {mce-short}. 

[#agent-not-removed]
=== Agent is not removed if the local-cluster is disabled
//2.13:ACM-17221

> The agent will not be removed from the hub automatically if you disable the local-cluster by setting the `disableHubSelfManagement` option to *true* in the `MultiClusterHub` resource. This issue appears only within 60 minutes after you upgrade the hub to version 2.13.

Reviewer: ```suggestion
The agent is not be removed from the hub automatically if you disable the `local-cluster` by setting the `disableHubSelfManagement` option to `true` in the `MultiClusterHub` resource. This issue appears only within 60 minutes after you upgrade the hub cluster to version 2.13.
```

> To work around this issue and remove the agent:

Reviewer: ```suggestion
To work around the issue and remove the agent, complete the following steps:
```

. Delete the `klusterlet` resource manually:
+
[source,terminal]
----

> *oc delete klusterlet klusterlet*

Reviewer 2: Please remove the unnecessary `*` at the beginning and end of the command.

Author: hi @elgnay . It will render the command in the bold font - according to [Conventions for commands and outputs](https://gitlab.cee.redhat.com/red-hat-enterprise-linux-documentation/rhel-8-docs/-/wikis/Conventions-for-commands-and-outputs).

Reviewer: We don't bold commands in ACM docs, I would just remove the bold

Author: fixed.

----

. If the `klusterlet` resource does not exist, delete the agent and add-on namespaces:
+
[source,terminal]
----

> *oc delete namespace --ignore-not-found open-cluster-management-agent open-cluster-management-agent-addon*

Reviewer 2: Please remove the unnecessary `*` at the beginning and end of the command.

Author: hi @elgnay . It will render the command in the bold font - according to [Conventions for commands and outputs](https://gitlab.cee.redhat.com/red-hat-enterprise-linux-documentation/rhel-8-docs/-/wikis/Conventions-for-commands-and-outputs).

Reviewer: Same comment as further up

Author: fixed
